styles.css

```diff
@@ -506,3 +528,398 @@ input[type="email"]:focus {
   border-width: 5px;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] **I can't continue reviewing until you fix this part**, please 🙏🏽 make sure that `@media screen and (min-width: 768px)` is working as expected:

https://user-images.githubusercontent.com/58642949/193282375-4937bbcc-8d50-4edd-abcf-ae03e9ba88f9.mp4

Reply:
@franclobo Please I don't understand why it is showing like this on your screen. In mine, it works perfectly fine.